File: client/app/src/main/java/com/rh/heji/data/db/BillWithImage.java
package com.rh.heji.data.db;

import androidx.room.Embedded;
import androidx.room.Relation;

import java.util.List;

/**
 * Date: 2020/9/15
 * Author: 锅得铁
 * #
 */
public class BillWithImage {
    @Embedded
    public Bill bill;

    @Relation(
            parentColumn = "_bid",
            entityColumn = "_id"
    )
    public List<Image> images;
}

File: client/app/src/main/java/com/rh/heji/data/db/BillWithImage.kt
package com.rh.heji.data.db

import androidx.room.Embedded
import androidx.room.Relation

/**
 * Date: 2020/9/15
 * Author: 锅得铁
 * #
 */
data class BillWithImage(
    @JvmField
    @Embedded
    var bill: Bill? = null,

    @JvmField
    @Relation(parentColumn = "bill_id", entityColumn = "_bid")
    var images: List<Image>? = null
) {

}